packages/eslint-plugin/src/rules/ban-context-export/index.js

```diff
@@ -5,8 +5,11 @@ const minimatch = require('minimatch');
 const createRule = require('../../utils/createRule');
 const { getTypeServices } = require('../../utils/type-services');
 
-/** @typedef { import('@typescript-eslint/utils').TSESTree.VariableDeclarator } VariableDeclarator*/
-/** @typedef { import('@typescript-eslint/utils').TSESTree.ExportSpecifier} ExportSpecifier */
+/**
+ * @import { TSESTree } from '@typescript-eslint/utils'
```

Comment:
this was not necessary for the actual TS migration right ? it's a very welcomed addition indeed but next time we should do this in a separate PR

Reply:
Yep, initially I wanted to make it in a separate PR